docs/source/howto/aforms.rst

```diff
@@ -0,0 +1,56 @@
+.. _aforms:
+
+How to extract AcroForm fields from a PDF using Python and PDFMiner
+********************************
+
+Before you start, make sure you have :ref:`installed pdfminer.six<install>`.
+
+The second thing you need is a PDF with AcroForms (as found in PDF files with fillable forms or multiple choices).
+
+It won't work for XFA forms.
+
+.. code-block:: python
+
+    from pdfminer.pdfparser import PDFParser
+    from pdfminer.pdfdocument import PDFDocument
+    from pdfminer.pdftypes import resolve1, PDFObjRef
+    from pdfminer.psparser import PSLiteral, PSKeyword
+    from pdfminer import utils
+    
+    data = {}
+    
+    with open(file_path, 'rb') as fp:
+        parser = PDFParser(fp)
+        
+        doc = PDFDocument(parser)
+        res = resolve1(doc.catalog)
```

Comment:
Maybe add a paragraph about what the `doc.catalog` is (because I don't know :p ).

Reply:
How do I add stuff to or modify a PR? 
(I can add a paragraph that explains doc.catalog later today)

Reply:
You can simply add new code, add a new commit, and push to your fork's `develop` branch (since that's what you started this PR from), and this PR will update.

This PR will always be the difference between your `develop` branch and the pdfminer one.

Reply:
Nice, thanks for the info.